doc/nep1.rst

```diff
@@ -280,3 +280,6 @@ Conventions for multi-line statements and expressions
   .. code-block:: nim
     startProcess(nimExecutable, currentDirectory, compilerArguments
                  environment, processOptions)
+
+- Use `a..b` instead of `a .. b`, except when `b` contains an operator, e.g `a .. -3`
```

Comment:
No eggs https://github.com/nim-lang/Nim/pull/15906#issuecomment-724879032

Please use `example`, easy for non-English native people, explicit, not ambiguous.

Reply:
done